### ISSUE TITLE
Create ACCESSIBILITY.md file - accessibility policy for the community - drawing from the Slack policies: https://hackmd.io/@turingway/ttw-slack-intro

### ISSUE THREAD

**Issue by aleesteele** [alan-turing-institute/the-turing-way#3145]

**Comment by aleesteele** [comment]
Notes & citations:
- Ongoing notes and citations: https://pad.sfconservancy.org/p/ttw-accessibility-md
- Accessibility Working Group Chapter Draft by @AlexandraAAJ: https://hackmd.io/WA3x623ZSCK410yvPZJV-w?both
- https://silviacanelon.com/accessibility/ - Accessibility page example
- https://hackmd.io/WA3x623ZSCK410yvPZJV-w?both
- Slack policies: https://hackmd.io/aUAcIE3VQvCYqgyGFbTO9g?both
- #2942 -

**Comment by aleesteele** [comment]
Examples of accessibility policies (list developed with @LizHareDogs) 
- https://digitalaccessibility.uoregon.edu/about/whatisda
- https://www.nyc.gov/assets/oti/html/nyc-core-framework/index.html
- https://digital.gov/resources/introduction-accessibility/
- https://silviacanelon.com/accessibility/
- https://github.com/govuk-react/govuk-react/blob/main/ACCESSIBILITY.md

**Comment by aleesteele** [comment]
We are in the final reviews of this policy - to be merged at the last Collaboration Cafe of the year! 🎉

**Comment by aleesteele** [comment]
@all-contributors please add @AlexandraAAJ for a11y

**Comment by aleesteele** [comment]
@all-contributors please add @LizHareDogs for review

**Comment by aleesteele** [comment]
@all-contributors please add @Arielle-Bennett for review and a11y

**Comment by aleesteele** [comment]
@all-contributors please add @harrietrs for review

**Comment by aleesteele** [comment]
@all-contributors please add @harrietrs for a11y

**Comment by aleesteele** [comment]
@all-contributors please add @JFormoso for review and a11y

**Comment by aleesteele** [comment]
@all-contributors please add @LelleDi for review and a11y

**Comment by aleesteele** [comment]
@all-contributors please add @LelleDi for event organizing

**Comment by aleesteele** [comment]
@all-contributors please add @malvikasharan for review and a11y

**Comment by aleesteele** [comment]
(now this issue is becoming an update contributors list!) 
@all-contributors please add @aleesteele for documentation, talk, tutorials, maintenance, video

**Comment by aleesteele** [comment]
@all-contributors please add @Preshh0 for a11y and review

**Comment by aleesteele** [comment]
@all-contributors please add @RichardJActon for review and a11y

**Comment by aleesteele** [comment]
@all-contributors please add @SaraVilla for review and a11y

**Comment by aleesteele** [comment]
@all-contributors please add @SaranjeetKaur for review and a11y

**Comment by aleesteele** [comment]
@all-contributors please add @BrainonSilicon for a11y, content and review

**Comment by aleesteele** [comment]
@all-contributors please add @trallard for review and a11y
